Source/inv.cpp

```diff
@@ -159,12 +159,12 @@ void InitInv()
 	drawsbarflag = false;
 }
 
-static void InvDrawSlotBack(const CelOutputBuffer &out, int x, int y, int w, int h)
+static void InvDrawSlotBack(const CelOutputBuffer &out, Point position, Size size)

```

Comment:
Do you plan to switch this to `Rect slot` at a later point and why not now?

Reply:
Yep. All Point+Size instances or even Point+some constant will invariably turn into rectangles.

Reply:
As for why, i like doing things incrementally. Check the next PR for rectangle stuff

Reply:
Not sure an in between pos+size stage makes it more incremental, it just feels like more churn, but let me look a bit at #2131 and maybe that makes it more clear/digestible.

Reply:
The many commits with rewrites of the same code makes it hard for me to see what the end result is for the various parts of the code. Could you at least add the commits needed for making Post+Size in to Rect to this PR (but not Contains or the changes to the mouse related code) so that I can see it in a combined view.

Reply:
> The many commits with rewrites of the same code makes it hard for me to see what the end result is for the various parts of the code.

There is no such thing as "end result". This same code will be touched upon again in the future with further refactoring. You can check the total result of the PR by just clicking the "Files changed" tab if you don't think the incremental view is worthwhile.

If you want to see the result of the application of this plus #2131, the latter currently contains this change in it (it will be rebased out once this is merged) so you can just look into the "Files changed" tab of #2131 instead.

> Could you at least add the commits needed for making Post+Size in to Rect to this PR (but not Contains or the changes to the mouse related code)

That doesn't make sense since we are not using `Rectangle` in here though. Rectangle logic starts being used on #2131 .

Reply:
> There is no such thing as "end result".

Sure, was the best way I could figure to phrase it at the time. Maybe "series target" would be a better one.

> You can check the total result of the PR by just clicking the "Files changed" tab

I know. But I'm asking for a PR that has More then #2130 and less then #2131, there is no tab for that. I think we already concluded that we review things a bit differently. If you can 
add c64942b276f7b23508e4684176d7c2db53a35431, 3f918c6e9bcaf61a970e7109ae31d0c7466e744c, 89793cef1916d957a00620dc10f79f9adcf71e14 and a7a26602e126408397b390f23ac5c364a0011442 to this PR it would make reviewing this series a lot easier for me. If not then I'll have to do it manually later at a later time when time permits.

Reply:
Commits added.